src/components/Review.jsx

```diff
@@ -26,6 +30,23 @@ const Review = () => {
 		e.preventDefault();
 	};
 
+	const handleReview = (num) => {
+		setReview(() => {
+			const copyObj = Object.assign({}, review);
+			copyObj.votes = copyObj.votes + num;
+			return copyObj;
+		});
+		return patchReview(review.review_id, num)
+			.catch((err) => {
+				setReview(() => {
+					const copyObj = Object.assign({}, review);
+					copyObj.votes = copyObj.votes - num;
+					return copyObj;
+				});
+				setErr(true);
+			})
+			.then(() => {});
```

Comment:
If you're not performing any logic inside this .then, it can be removed entirely.